src/ui/server/helpers/object/index.test.ts

```diff
@@ -25,6 +25,14 @@ describe('server/helpers/object', () => {
         expect(result).toEqual(false);
       });
     });
+
+    describe('when there is no object', () => {
```

Comment:
Test description says no object, but an object is provided 😝

Reply:
removed the test, since eslint kicks off